controllers/node_module_mapper_test.go

```diff
@@ -67,11 +89,17 @@ var _ = Describe("FindModulesForNode", func() {
 			},
 		}
 
-		nmm := controllers.NewNodeModuleMapper(
-			fake.NewClientBuilder().WithScheme(scheme).WithObjects(&mod1, &mod2, &node).Build(),
-			logr.Discard(),
+		client := client.NewMockClient(ctrl)
+		client.EXPECT().List(context.TODO(), gomock.Any(), gomock.Any()).DoAndReturn(
+			func(ctx context.Context, list *v1alpha1.ModuleList, _ interface{}) error {
```

Comment:
This `func` MUST have 3 arguments, otherwise the testing library, by reflection, will raise this error:
```
Test Panicked
  In [It] at: /usr/local/go/src/reflect/value.go:398
  reflect: Call with too few input arguments
```

So, In [the implementation](https://github.com/qbarrand/oot-operator/pull/69/commits/9983d58efefa17c8b8d350283c266144228a7c44#diff-2e7facd3081a7f636bc0a916dea254e506b6d27488e7e29d1046f23037c0ae27R36) I needed to add a third parameter `nil` to satisfy this requirement.

It's not nice when the test forces us to code something unnecessary. Thoughts on this? 
@ybettan @yevgeny-shnaidman